docker/README.md

```diff
@@ -23,7 +17,7 @@ The Dockerfiles included in this project are for openSUSE:
 Our intent was to create universal `webui` and `worker` containers and move all data storage and configurations to third container,
 called `openqa_data`. `openqa_data` is so called [Data Volume Container](http://docs.docker.com/userguide/dockervolumes/#creating-and-mounting-a-data-volume-container)
 and is used as database, results and configuration storage. During development and in production, you could update `webui` and `worker` images
-but as long as `openqa_data` is intact, you don't lose any data.
+but as long as `openqa_data` is intact, you don't lochanged in the docker-compose.yaml for the service haproxy. For instance tose any data.
```

Comment:
This changes is still wrong ("lochanged").

Reply:
Yes, a paste error. Some text from any other place has pasted here. Fixed